common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java

```diff
@@ -340,7 +355,56 @@ public Connection connectToServer(ServerInfo serverInfo) {
         }
         return null;
     }
-    
+
+    private ManagedChannelBuilder buildChannel(String serverIp, int port, Optional<SslContext> sslContext) {
+        if (sslContext.isPresent()) {
+            return NettyChannelBuilder.forAddress(serverIp, port)
+                    .negotiationType(NegotiationType.TLS)
+                    .sslContext(sslContext.get());
+
+        } else {
+            return ManagedChannelBuilder
+                    .forAddress(serverIp, port).usePlaintext();
+        }
+    }
+
+    private Optional<SslContext> buildSslContext() {
+        TlsConfig tlsConfig = clientConfig.tlsConfig();
+        if (!tlsConfig.getEnableTls()) {
+            return Optional.absent();
+        }
+        try {
+            SslContextBuilder builder = GrpcSslContexts.forClient();
+            builder.sslProvider(SslProvider.OPENSSL);
+            if (StringUtils.isNotBlank(tlsConfig.getProtocols())) {
+                builder.protocols(tlsConfig.getProtocols().split(","));
```

Comment:
default protocol is？

Reply:
if we no specified  protocols, client and server will negotiator when tls handshake.